### ISSUE TITLE
Expand French data queries

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-Data#288]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-Data/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-Data's [Code of Conduct](https://github.com/scribe-org/Scribe-Data/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue would look into expanding the [src/scribe_data/language_data_extraction/French](https://github.com/scribe-org/Scribe-Data/blob/main/src/scribe_data/language_data_extraction/French) files with as much data as are possible from the current data on Wikidata. We can use code for getting data from other languages, and from there we can check French data on Wikidata for what conjugations are available. We can then expand the query with optional selections of certain forms as is done in other SPARQL queries. The query can be tried on the [Wikidata Query Service UI](https://query.wikidata.org/) during development :)

Data types to include:
- [x] Nouns
- [x] Verbs
- [x] Adjectives
- [x] Adverbs
- [x] Prepositions
- [x] Emoji keywords

Have marked those that are already done or have PRs open, and we can work on the others 😊 If the data type can't work, then we can move to the others and open up specific issues later :)

### Contribution

Happy to support and review PRs for this as needed! Let us know if you'd like to work on one of the data types!

**Comment by VNW22** [comment]
heyy, i would like to work on this issue.

**Comment by andrewtavis** [comment]
Great, @VNW22! Do you want to take adjectives and adverbs for now, and then if prepositions is still available you could do that too once the PR for the others is merged? :)

**Comment by VNW22** [comment]
Sounds good! I’m happy to take on adjectives and adverbs for now. Once the PR is merged, I’d be glad to handle prepositions if it's still available. Thanks!

**Comment by andrewtavis** [comment]
Thank you, @VNW22!

**Comment by andrewtavis** [comment]
Closed by #299 🚀 Thanks much for the work here, @KesharwaniArpita!

Please let us know if there's another issue you'd like to pick up, @VNW22 :)

**Comment by VNW22** [comment]
Thanks for the update, @KesharwaniArpita!
@andrewtavis As for checking another language, I’m happy to take on that task!  no problem transferring the progress from here to a new issue.
